src/DynamoCoreWpf/Views/Core/NoteView.xaml

```diff
@@ -31,51 +31,67 @@
         </Grid.ContextMenu>
         <Canvas Background="{x:Null}">
         </Canvas>
-
-        <TextBlock
-          Foreground="White"
-          Text="{Binding Path=Text, Mode=OneWay}"
-          Grid.Column="0" Grid.Row="0" FontFamily="Trebuchet" 
-            FontSize="12" 
-            Padding="10" TextWrapping="Wrap">
-            
-          <TextBlock.RenderTransform>
-            <TranslateTransform X="0" Y="-0.4" />
-          </TextBlock.RenderTransform>
-            
-        <TextBlock.Style>
-            <Style TargetType="TextBlock">
-
-                <Style.Triggers>
-                    <DataTrigger Binding="{Binding Path=IsSelected, Mode=OneWay}" Value="True">
-                        <Setter Property="Background" Value="LightBlue"/>
-                    </DataTrigger>
-                    <DataTrigger Binding="{Binding Path=IsSelected, Mode=OneWay}" Value="False">
-                        <Setter Property="Background" Value="#EEE"/>
-                    </DataTrigger>
-                </Style.Triggers>
-
-            </Style>
-        </TextBlock.Style>
-            
-        <TextBlock.Effect>
-            <DropShadowEffect
-                ShadowDepth="0"
-                Color="#222"
-                Opacity="0.25"
-                BlurRadius="9.0" />
-        </TextBlock.Effect>
-            
-        </TextBlock>
         
+        <TextBox
+            Name ="noteTextBox"
+            Grid.Column="0" Grid.Row="0"
+            Text="{Binding Path=Text, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" 
+            FontFamily="Trebuchet" 
+            FontSize="12"
+            Foreground="#222"
+            AcceptsReturn="True"
+            AcceptsTab="True"
+            Background="#EEE"
+            Padding="10" TextWrapping="Wrap"
+            LostFocus="noteTextBox_LostFocus"
+            PreviewKeyDown="noteTextBox_PreviewKeyDown"
+            IsEnabled="{Binding Path=IsOnEditMode, Mode=OneWay}"
```

Comment:
@martinstacey Do you mind moving the inline style definition into the central location `DynamoModern.xml`?

Reply:
ok addressed in c7dd3c7